src/cleanlab_tlm/utils/tlm_calibrated.py

```diff
@@ -21,9 +21,95 @@
 
 if TYPE_CHECKING:
     from collections.abc import Sequence
-
     from cleanlab_tlm.internal.types import TLMQualityPreset
 
+def _get_skops():
+    """Lazy import for skops to avoid unnecessary dependency."""
+    try:
+        import skops.io
+        return skops.io
+    except ImportError:
+        raise ImportError(
+            "The skops package is required for model serialization. "
+            "Please install it with: pip install skops"
+        )
+
+def save_tlm_calibrated_state(model: 'TLMCalibrated', custom_eval_options: Dict[str, Any], filename: str) -> None:
+    """Save fitted TLMCalibrated model state to file.
+    
+    Args:
+        model: A fitted TLMCalibrated model instance
+        custom_eval_options: Dictionary containing custom evaluation options
+        filename: Path where the model state will be saved
+    
+    Raises:
+        sklearn.exceptions.NotFittedError: If the model has not been fitted
+        ImportError: If skops package is not installed
+    """
+    try:
+        from sklearn.utils.validation import check_is_fitted
+    except ImportError:
+        raise ImportError(
+            "Cannot import scikit-learn which is required to use TLMCalibrated. "
+            "Please install it using `pip install scikit-learn` and try again."
+        )
+    
+    # Verify model is fitted
+    check_is_fitted(model._rf_model)
```

Comment:
I'd print a more explicit error message for this here, can copy something similar to this: https://github.com/cleanlab/cleanlab-tlm/blob/main/src/cleanlab_tlm/utils/tlm_calibrated.py#L126-L131